src/classes/LineItemTriggerHandler0088.cls

```diff
@@ -0,0 +1,131 @@
+public with sharing class LineItemTriggerHandler0088 {
+		
+  
+
+	
+	/** 
+	This handler methods is to be used on all relevant triggers events.
+		It updates purchased amount on orders and update top product categories.
+		Logic is fully bulkified. */		
+	public static void handle(List<Line_Item__c> items) {
+		
+		// just a comment for a fake code review
+
+		// collect related orders
+		Set<Id> orderIds = new Set<Id>();
+		for (Line_Item__c i : items) {
+			orderIds.add(i.Order__c);
+			update i;
```

Comment:
### Development Essentials: Data access or manipulation in loops
#### Finding
This data manipulation is inside a loop. Please use a collection to store relevant data, and move any database manipulation outside the loop.

#### Why is this a problem?
Salesforce enforce a limit on the number of SOSL and DML statements per transaction. Reading or writing data inside loops is highly likely to hit such limits.

#### Help and documentation

* [Uncontrolled Resource Consumption (CWE)](https://cwe.mitre.org/data/definitions/400.html) 
* [Avoid SOQL Queries or DML statements inside FOR Loops (Developer Documentation)](https://developer.salesforce.com/page/Apex\_Code\_Best\_Practices\#Best\_Practice\_.232:\_Avoid\_SOQL\_Queries\_or\_DML\_statements\_inside\_FOR\_Loops)